makefile

```diff
@@ -1,43 +0,0 @@
-# Usage:
```

Comment:
@eguiraud I've always kept the makefile for "old fashion compatibility". Everyone who's used to work with Fortran is used to a makefile. While things like setup.py and pyproject.toml change over time, the makefile is the simplest and most robust thing... but if you think that it should be removed, then so be it.

Reply:
`make aurora` compiles the fortran into a shared library, but it does not produce a working python environment (it does not install the dependencies, it does not check that the compiled shared object can be imported from python, etc.).

it would make sense to also have a makefile if aurora was also meant to be used as a standard fortran module: then if you want a python package you use `pip install`, if you want a fortran library you use `make aurora`. my understanding was that aurora is only meant to be used as a python package, so it should use normal python-packaging systems.